src/components-examples/cdk/drag-drop/cdk-drag-drop-table/cdk-drag-drop-table-example.html

```diff
@@ -0,0 +1,38 @@
+<mat-table #table [dataSource]="dataSource" class="mat-elevation-z8" cdkDropList (cdkDropListDropped)="drop($event)"
+  cdkDropListData="dataSource" [cdkDropListDisabled]="dragDisabled">
+  <!-- Position Column -->
+  <ng-container matColumnDef="position" sticky>
+    <mat-header-cell *matHeaderCellDef> No. </mat-header-cell>
+    <mat-cell *matCellDef="let element">
+      <mat-icon class="example-drag-cursor" (mousedown)="dragDisabled = false;">reorder</mat-icon>

```

Comment:
Why is dragging disabled initially? When I was trying it out, it felt a bit buggy because it wouldn't start dragging the first time. I think that we should just remove the `dragDisabled` binding.

Reply:
I wanted to enable dragging only when the user clicks the drag icon, but I see verry laggy. I'll just remove it.